servers/src/main/java/tachyon/worker/block/meta/StorageDir.java

```diff
@@ -373,7 +373,7 @@ public void removeTempBlockMeta(TempBlockMeta tempBlockMeta) throws NotFoundExce
   }
 
   /**
-   * Changes the size of a temp block or throws IOException.
+   * Changes the size of a temp block or throws Exception.
    *
```

Comment:
Since there is `@throws ...` expression, do you think these `or throws .....` is redundant ?